gatling-s3-reporter/README.md

```diff
@@ -0,0 +1,21 @@
+# Gatling S3 reporter
+
+Generate report page from Gatling logs.
+https://gatling.io/docs/2.2/
+
+Gatling logs must be stored in S3 directory, with filename `*.log`.
+Gatling report page is created on directory where logs are stored.
+
+## Usage
+
+### basic example
+
+```
+$ docker run -it --rm -v ~/.aws/credentials:/root/.aws/credentials:ro -e S3_GATLING_BUCKET_NAME=[s3_bucket_name] -e S3_GATLING_RESULT_DIR_PATH=[s3_directory] chatwork/gatling-s3-reporter:latest
```

Comment:
@kamegu 
Isn't it better to mount the `${PWD}/scenario` on `/opt/work/scenario` and specify it with `chatwork/gatling-s3-reporter:latest -sf scenario/`?

Reply:
Thanks for advice.
For this image, running scenario is out of scope.
I am also considering how to run scenario easily with this image or other way.